staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -45,8 +45,6 @@ import (
 type Reflector struct {
 	// name identifies this reflector. By default it will be a file:line if possible.
 	name string
-	// metrics tracks basic metric information about the reflector
-	metrics *reflectorMetrics
```

Comment:
Instead of deleting the metrics, we probably should fix the metrics, see workqueue [metrics](https://github.com/kubernetes/kubernetes/blob/836b901658557ee18936d3690dc9c48fc588eba8/staging/src/k8s.io/client-go/util/workqueue/metrics.go#L236) as an example.

@logicalhan IIRC there was a plan to overhaul the metrics in k8s. I don't know if that plan will affect the fix in this package. Could you advise? Thanks.

Reply:
Actually these metrics are already deleted. https://github.com/kubernetes/kubernetes/pull/74636, I believe I missed these references in my cleanup. My apologies.

Reply:
It is staticcheck works.
Maybe we will never miss any cleanup anymore after this pr.